mirgecom/navierstokes.py

```diff
@@ -521,7 +543,10 @@ def ns_operator(dcoll, gas_model, state, boundaries, *, time=0.0,
         dd=dd_vol)
 
     # Add corresponding inviscid parts if enabled
-    if inviscid_terms_on:
+    # Note that this is the default, and highest performing path.
```

Comment:
Not enough.   The choices are more varied than ESDG or not.   We can use local vs. external operator inviscid terms.  If the operator is separate, then it can be the Euler operator, or the ESDG operator (or a different one given by the user).

The old, default path, is to compute inviscid terms locally and hit only once with the divergence operator.   That's the current default.